### PR TITLE
Fix Aliyun DNS issue which change env name doesn't trigger DNS update

### PR DESCRIPTION
**Pull request by loganhz**
https://github.com/rancher/rancher/issues/10077

### CODE REVIEW

**Comment by deniseschannon** [comment]
@loganhz can you separate it into 2 commits, one for the copy of the previous folder and the other commit with the changes? it makes it easier to review.

**Comment by loganhz** [comment]
@deniseschannon done

File: infra-templates/alidns/2/rancher-compose.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # notemplating
 .catalog:
   name: "Alibaba Cloud DNS"
-  version: "v0.2.0"
+  version: "v0.3.0"
   description: "Rancher External DNS service powered by Alibaba Cloud"
   minimum_rancher_version: v1.6.8-rc1
```
Comment (deniseschannon) [inline infra-templates/alidns/2/rancher-compose.yml]: can we update this to what will be the next release as we will only be testing against that version? 

`v1.6.12-rc1`

**Comment by deniseschannon** [comment]
LGTM

**Pull request merged**